lms/djangoapps/courseware/models.py

```diff
@@ -189,7 +188,6 @@ class XModuleStudentInfoField(XBlockFieldBase):
     """
     Stores data set in the Scope.preferences scope by an xmodule field
     """
-
     class Meta(object):  # pylint: disable=missing-docstring
         unique_together = (('student', 'field_name'),)
 
```

Comment:
had Used `objects=CallStackManager()` and removed it after the use. deleted the line in the end.